transit-ruby.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
     public_key_path = nil  # not yet supported
   else
     spec.files    = files - jruby_files
-    spec.add_dependency "oj",                             "~> 2.18"
+    spec.add_dependency "oj",                             "~> 3.10"
```

Comment:
Should it be "~> 3.10", ">= 3.10.16" to prevent an accident installing of broken versions mentioned in the linked issue?

Reply:
I copied that from their fork. But yeah, let's do that.

```suggestion
    spec.add_dependency "oj",                             "~> 3.10", ">= 3.10.16"
```